packages/page-claims/src/statements/regular.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017-2020 @polkadot/app-claims authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+
+// Copy-pasted from https://statement.polkadot.network/regular.html
+function RegularStatement (): React.ReactElement {
+  return (
+    <>
+      <p className='bold_text'>Statement</p>
```

Comment:
This really is messy like this. Should be md.

Reply:
(And needs to be under statements/polkadot - will make a comment around that is the final overall later)

Reply:
The W3F decided to use a html instead of a md. We don't actually have the md anymore. Would rather stay as close as what the W3F gives us.